.github/workflows/workflow.yml

```diff
@@ -0,0 +1,32 @@
+name: Continous integration tests
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+concurrency:
+  group: build-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+env:
+  NODE_VERSION: 17.x
```

Comment:
:+1: on 17. I normally include the LTS version too (in this case I think Node LTS is 16?).

Reply:
Good catch! Will add this.